### ISSUE TITLE
Switching Workspaces Using Trading Farms With Virtual Machines

### ISSUE THREAD

**Issue by EduRemix** [Superalgos/Superalgos#270]
**Is your feature request related to a problem? Please describe.**
I am currently running the Backend of Superalgos on a Virtual Machine and the Frontend on my PC locally.

I do this as follows:
1) I leave the Backend running on the VM with the browser closed.
2) I run Superalgos on my PC locally.
3) In the Workspace, within the Network Node configuration, I put the IP of my VM.

It is the concept of a trading farm as is explained in the documentation, but with a VM.
It works great, and I love it!
The only problem I have is that when I change the Workspace locally all VM processes are shut down.

**Describe the solution you'd like**
I understand that it is necessary to shut down all processes at the local machine level when switching from Workspace, but I think that it would be convenient if changing the Workspace does not alter the processes that are running in the VM.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by 9808us** [comment]
I have the same problem. I would like to have a workspace dedicated to the VM and one (different) dedicated local one. The problem is that when you switch workspace, everything get stopped.
So, for the moment, waiting for a solution, I have done this:

- I have created two separate directories with 2 Superalgos installations, let's call them SuperVM and SuperLocal
- Launch Superalgos from the SuperVM directory and create a workspace dedicated to the VM
- When everything is set, stop all operations and close Superalgos (front + back end)
- With an editor I replaced any reference to localhost with the IP address of my master machine (192.168.0.2 for example)
- From the SuperVM dir Launch Superalgos and check that the local IP address is set in the browser and no longer localhost
- Set up all I need, start a paper/live session
- Close everything (My VM is still running in the cloud)

Now if I use the SuperLocal directory and launch Superalgos with default workspace, it is a distinct address for the browser (it reports localhost as default) and in this way I can set a different workspace for local use only

So now if I launch Superalgos from dir SuperVM, then I have a workspace set up for VM (note that mining data is on the VM so no extra space needed on my PC)
If I launch Superalgos from dir SuperLocal then I see my local workspace
